source/Color.h

```diff
@@ -25,6 +25,14 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 // zero the color's components will be added to whatever is underneath them.
 class Color {
 public:
+	// All the accessibility color filters, for easier debugging.
+	enum class Filter : int {
```

Comment:
this goes in preferences.h though, just look at how it is done with the other settings